extensions/Android.mk

```diff
@@ -13,6 +13,7 @@ LOCAL_SRC_FILES := \
 assets-manager/Manifest.cpp \
 assets-manager/AssetsManagerEx.cpp \
 assets-manager/CCEventAssetsManagerEx.cpp \
+$(LOCAL_PATH)/../cocos/base/CCAsyncTaskPool.cpp \
```

Comment:
Why to add $(LOCAL_PATH)?

Reply:
thanks for this check, I am wondering this.

`CCAsyncTaskPool ` is located at `cocos/base` folder, and the symbols never used in __libcocos2d.a__, it only used in `AssetsManagerEx.cpp` . 

Firstly I added `CCAsyncTaskPool.cpp` file into `cocos/Android.mk`, when link process, `undefined reference to` occurs. And then, I try to add it into `extensions/Android.mk`.  it works.

Reply:
I think if `CCAsyncTaskPool.cpp` was only used by __extensions__, we should put it into `extensions` folder. Now it located at `cocos/base`, case that files in `extensions` depend on `cocos` , files in `cocos` depend on `extensions`